benchto-driver/src/main/java/io/trino/benchto/driver/Benchmark.java

```diff
@@ -36,6 +36,7 @@
     private String environment;
     private List<Query> queries;
     private int runs;
+    private int queryRuns;
```

Comment:
I think it's confusing what `runs` and `queryRuns` means. Why `runs` are not sufficient?
Should we add execution schema (e.g. `SEQUENTIAL` -> `q1, q1, q1, ...`, `carousel` -> `q1, q2, q3, q1, q2, q3`) enum instead?

Reply:
This is similar approach to my first approach. Personally I prefer this approach.
https://github.com/trinodb/benchto/pull/60

Reply:
We could rename `runs` to `benchmarkRuns` but it is still confusing on my eye.